README.md

```diff
@@ -30,6 +34,13 @@ while using your knowledge to write readable, maintainable code.
 
 **Note:** [While the example][example] shows one working application of TaskLister Lite™️, yours can (and is encouraged to!) look however you like.
 
+## Squelch a Default Action with `Event.preventDefault`
+
+The deliverables require you to handle an event in a form based an clicking a submit button. You're going to wind to listen, on the `<form>` element for a `submit` event. Form tags, by default, automatically redirect the browser to a new URL. This _is not_ the experience we want. We want to intercept the `submit` event, yes; but we want to _prevent_ that event from triggering the redirect because ***we*** want to update the DOM using JavaScript. In order to _prevent_ this _default_ behavior of the
```

Comment:
typo: `wind to listen`

Reply:
```suggestion
The deliverables require you to handle an event in a form based on clicking a submit button. You're going to need to listen for a `submit` event on the `<form>` element. 

By default, Form elements automatically submit the form, which redirects the browser to a new url. This _is not_ the experience we want to build in this lab. We want to _prevent_ that event from performing its _default_ behavior (submitting the form), because ***we*** want to update the DOM using JavaScript. In order to _prevent_ the _default_ behavior of the
```